library/src/main/java/com/novoda/downloadmanager/LiteFileName.java

```diff
@@ -4,12 +4,12 @@
 
     private final String name;
 
-    public static LiteFileName from(Batch batch, String fileUrl) {
+    static LiteFileName from(Batch batch, String fileUrl) {
         String name = batch + fileUrl + String.valueOf(System.nanoTime());
         return new LiteFileName(String.valueOf(name.hashCode()));
     }
 
-    public static LiteFileName from(String name) {
+    static LiteFileName from(String name) {
```

Comment:
LiteFileName? Reference to the package name that was changed above, "litedownloadmanager" ?

Reply:
There are a lot of Classes that are prefixed with 'Lite', we'll tackle them all at a later date 👍

Reply:
Yeah tbh I noticed it more as I went on, so understand now 👍